### PR TITLE
Storage account names are only a-z0-9

### PR DESCRIPTION
**Pull request by hunner**

### CODE REVIEW

File: spec/acceptance/storage_account_spec.rb [hunk 1]
```diff
@@ -1,10 +1,12 @@
 require 'spec_helper_acceptance'
 
-describe 'azure_storage_account when creating a storage account' do
+describe 'azure_storage_account when creating a storage account', :focus => true do
   include_context 'with a known name and storage account name'
   include_context 'destroy left-over created ARM resources after use'
 
   before(:all) do
+    @client = AzureARMHelper.new
+    @name = @client.get_simple_name(@name)
```
Comment (bmjen) [inline spec/acceptance/storage_account_spec.rb]: This line is confusing. @name doesn't appear to be initialized anywhere.


sidenote: Do names allow dashes and underscores?

Comment (bmjen) [inline spec/acceptance/storage_account_spec.rb]: -sidenote: just noticed your commit msg said that Storage Account names only allow alphanumeric chars.

Comment (hunner) [inline spec/acceptance/storage_account_spec.rb]: No dashes or underscores. @name is initialized in the spec helper, and I haven't refactored the specs to be more clear. You know how those things go...

Comment (bmjen) [inline spec/acceptance/storage_account_spec.rb]: hmm. The function for `get_simple_name` is in the spec_helper_acceptance, but it's not called anywhere...

Comment (hunner) [inline spec/acceptance/storage_account_spec.rb]: It's called on the line you're commenting on :P

**Pull request merged**